### ISSUE TITLE
Fatal Compile Error: Unparenthesized `a ? b : c ? d : e` is not supported.

### ISSUE THREAD

**Issue by artshevtsov** [E1379/speaking-url#1]
Compiler version: PHP 8.0.2

```
$su = new SpeakingUrl();
throws:
In SpeakingUrl.php line 89:
                                                                                                                                  
  Compile Error: Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)`
```

**Comment by webpouya** [comment]
Hi, I have the same problem
Can anyone solve this?

Compile Error: Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)`

in [public_html/language/fa-IR/fa-IR.localise.php (line 115)

cod:
if (strpos($return, self::MONTH_LENGTH) !== false) {
			$return = str_replace(self::MONTH_LENGTH, $m < 7 ? 31 : $m < 12 ? 30 : self::leap_persian($y) ? 30 : 29 , $return);
		}
		if (strpos($return, self::YEAR_ABBR) !== false) {
			$return = str_replace(self::YEAR_ABBR, sprintf("%02d",$y % 100), $return);
		}
		if (strpos($return, self::YEAR_NAME) !== false) {
			$return = str_replace(self::YEAR_NAME, $y, $return);
		}
		if (strpos($return, self::AM_LOWER) !== false) {
			$return = str_replace(self::AM_LOWER, $this->format('a',$local)=='pm' ? 'ب ظ' : 'ق ظ', $return);
		}
		if (strpos($return, self::AM_UPPER) !== false) {
			$return = str_replace(self::AM_UPPER, $this->format('a',$local)=='pm' ? 'ب ظ' : 'ق ظ', $return);

**Comment by E1379** [comment]
This is fixed in 0.0.2. 
@webpouya Your error is not caused by this package.
